asv_setup/config/robots/freya.yaml

```diff
@@ -58,4 +58,18 @@
     internal_status:
       psm_read_rate: 2.0 # [readings/second], Recomended 2.0 readings per second OBS! Try keeping under 2.0 as otherwise I2C comunication to Power Sense Module might become bad as psm needs time to sample data
 
+    acoustics:
+      frequencies_of_interest: # MUST be 20 elements [FREQUENCY [Hz], FREQUENCY_VARIANCE [Hz]...] Recoended: Frequency 1 000 - 50 000 Hz, Variance 1 000 - 10 000
+       [10000, 1000,
+        50000, 3000,
+        0, 0,
+        0, 0,
+        0, 0,
+        0, 0,
+        0, 0,
+        0, 0,
+        0, 0,
+        0, 0]
+      data_logging_rate: 1.0 # [logings/second], Recomended: 1.0 logings per second
+
```

Comment:
is it not possible to just use an array of tuples? Idk asking for a friend

Reply:
I might look into it later as ROS2 didn't provide off the shelf the ability to convert a list of tuples. Might be a feature later on, maybe a add on library or a list of strings that have tuples, will see jesjes 👍